### PR TITLE
feat(TCOMP-2258): @Documentation to tooltips in uiSchema

### PR DESCRIPTION
**Pull request by undx**
https://jira.talendforge.org/browse/TCOMP-2258

### Requirements

* Any code change adding any logic **MUST** be tested through a unit test executed with the default build
* Any API addition **MUST** be done with a documentation update if relevant

### CODE REVIEW

File: component-server-parent/component-server/pom.xml [hunk 1]
```diff
@@ -186,6 +186,12 @@
       <version>${project.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.talend.sdk.component</groupId>
+      <artifactId>component-form-core</artifactId>
+      <version>1.49.0-SNAPSHOT</version>
```
Comment (ozhelezniak-talend) [inline component-server-parent/component-server/pom.xml:192]: project version?
instead of hardcoded

**Review by ozhelezniak-talend — APPROVED** [review]
lgtm. minor comment

File: component-server-parent/component-server/pom.xml (hunk 1 above)
Comment (undx) [reply component-server-parent/component-server/pom.xml:192]: arg, yep!

**Review by ozhelezniak-talend — APPROVED** [review]
lgtm

**Pull request merged**